src/main/kotlin/de/bund/digitalservice/useid/identification/TcTokenService.kt

```diff
@@ -0,0 +1,26 @@
+package de.bund.digitalservice.useid.identification
+
+import de.bund.digitalservice.useid.eidservice.EidService
+import de.bund.digitalservice.useid.utils.XmlHelper
+import de.governikus.autent.sdk.eidservice.tctoken.TCTokenType
+import mu.KotlinLogging
+import org.springframework.stereotype.Service
+import reactor.core.publisher.Mono
+
+@Service
+class TcTokenService(
+    private val eidService: EidService
+) {
+    private val log = KotlinLogging.logger {}
+    fun getTcToken(refreshAddress: String): Mono<TCTokenType> {
+        /* TODO: current implementation is a blocking operation
+        * We cannot just wrap a blocking call with a Mono or a Flux
+        * https://betterprogramming.pub/how-to-avoid-blocking-in-reactive-java-757ec7024676
+        * and also usage of Reactor's BlockHound is preferred
+        */
+        return Mono.just(eidService.getTcToken(refreshAddress))
+            .doOnNext {
+                log.info("created tcToken ${XmlHelper.marshalObject(it)}")
```

Comment:
I don't think we need a log here, especially not "info", since this route will be one of the most called ones.

Reply:
probably not for `success` case and I agree the XML helper is for debugging purpose, but in case `getTcToken` returns error (e.g server error), i think we need to log it for e.g Sentry? what do you think?

Reply:
Yes, good point, we need to log the response of the eIDServer in an error case (but still probably not `info`, maybe even higher?).

Reply:
I created a story for Sentry, because it will increase this PR and we should properly discuss what error we want to observe => https://digitalservicebund.atlassian.net/browse/USEID-312

I would remove this `log` and the `XMLHelper` for now.